kubernetes-deployment/src/chart/templates/ingress.yaml

```diff
@@ -30,13 +30,14 @@ spec:
   {{- if and .Values.ingress.className (semverCompare ">=1.18-0" .Capabilities.KubeVersion.GitVersion) }}
   ingressClassName: {{ .Values.ingress.className }}
   {{- end }}
+  {{- if .Values.ingress.host }}
   tls:
     - hosts:
         - {{ $host | quote }}
       secretName: {{ $tls }}
+  {{- end }}
   rules:
-    - host: {{ $host | quote }}
```

Comment:
Unfortunately, it won't be this simple. `host` is a required field - without it, the ingress controller won't know how to route the traffic.  I have a couple ideas - let's sync on this tomorrow.